canvas_modules/common-canvas/package.json

```diff
@@ -88,8 +88,9 @@
     "grunt-jsonlint": "2.1.3",
     "grunt-postcss": "0.9.0",
     "grunt-yamllint": "0.3.0",
-    "jest": "26.4.2",
+    "jest": "29.7.0",
     "jest-fetch-mock": "3.0.3",
+    "jest-fixed-jsdom": "^0.0.9",
```

Comment:
This should be a static value since this is a dev dependency.

Reply:
@matthoward366 I changed it. Please re-review.